### DISCUSSION TITLE
Will this still be supported?

### DISCUSSION THREAD

**Discussion by lbk9** [jonasbark/flutter_stripe_payment#298] [General]
The pub dev page for this widget now mentions that it has been discontinued. Will this widget still be functional? I get that there will no longer be updates made to it but will it still work for apps that already have the widget integrated?

**Comment by jonasbark** [comment]
The plugin is discontinued, not the widget itself. For the time being, I will continue to merge PRs with hotfixes but won't work on this plugin myself anymore.

**Reply by lbk9** [reply]
I understand, all I'm asking is that will I still be able to use this plugin to make payments with stripe or will I need to migrate to the new version?
